lib/ncr_dispatcher.rb

```diff
@@ -34,4 +34,10 @@ def on_proposal_update(proposal)
       end
     }
   end
+
+  def on_approver_removal(proposal, approvers)
+    approvers.each{|approver|
+      CommunicartMailer.notification_for_subscriber(approver.email_address,proposal,"removed").deliver_now
+    }
+  end
```

Comment:
Why not have this in the `Dispatcher`? Doesn't seem like NCR-specific functionality.